plugins/pagerduty/src/plugin.ts

```diff
@@ -51,7 +53,45 @@ export const EntityPagerDutyCard = pagerDutyPlugin.provide(
     name: 'EntityPagerDutyCard',
     component: {
       lazy: () =>
-        import('./components/PagerDutyCard').then(m => m.PagerDutyCard),
+        import('./components/EntityPagerDutyCard').then(
+          m => m.EntityPagerDutyCard,
+        ),
+    },
+  }),
+);
+
+export const homePlugin = createPlugin({
+  id: 'pagerduty-home',
```

Comment:
Why make a new plugin? The extension should just be provided by the pagerduty plugin